Mathlib/RingTheory/SimpleRing/Defs.lean

```diff
@@ -0,0 +1,25 @@
+/-
+Copyright (c) 2024 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang
+-/
+
+import Mathlib.RingTheory.TwoSidedIdeal.Lattice
+import Mathlib.Order.Atoms
+
+/-! # Simple rings
+
+A ring `R` is **simple** if it has only two two-sided ideals, namely `0` and `⟨1⟩`.
```

Comment:
Or `⊥` and `⊤` instead? Especially because `⟨1⟩` is not valid Lean syntax for a `TwoSidedIdeal`, right?